### ISSUE TITLE
Can we display list items in the wiki differently?

### ISSUE THREAD

**Issue by ckipp01** [XXIIVV/Webring#233]
Currently if you have a list under a category, it just separates them by comma and puts them all in one `<li>`. Do you think instead if there is a list we can make a them display a bit nicer and more separated?

Here is an example of what it currently looks like

![image](https://user-images.githubusercontent.com/13974112/62413193-4af65b00-b60d-11e9-8a00-b98769ccc694.png)
 
and with another nested `<ul>` it could look like this

![image](https://user-images.githubusercontent.com/13974112/62413232-9b6db880-b60d-11e9-852a-4ad262fc0f13.png)

**Comment by joshavanier** [comment]
Is that yours? I'd like to see how it's structured in the `.ndtl` file.

**Comment by ckipp01** [comment]
Yea, I have it structured like so
```
SPACE
  Keplerian elements : The traditional 6 orbital elements which are the parameters required to uniquely identify a specific orbit.
  Orbital eccentricity : Parameter that related to an astronomical object that determines the amount by which its orbit around another body deviates from a perfect circle.
  Orrery : A mechanical model of the solar system that illustrates or predicts the relative positions and motions of the planets and moons, usually according to the heliocentric model.
  Three laws of planetary motion discovered by Johannes Kepler
    The planets move in elliptical orbits with the Sun at one focus.
    The time necessary to traverse any arc of a planetary orbit is proportional to the area of the sector between the central body and that arc (the "area law").
    There is an exact relationship between the squares of the planets' periodic times and the cubes of the radii of their orbits (the "harmonic law").
```
and looking at @neauoire's site [here](https://wiki.xxiivv.com/#indental) it should be allowed since he shows it in the picture. It actually parses correctly into an array, but just isn't shown right.

I'm actually working on it right now. Should have a pr tonight to fix it.

**Comment by ckipp01** [comment]
Fixed with #235
